esmvaltool/recipes/hydrology/recipe_pcrglobwb.yml

```diff
@@ -2,16 +2,22 @@
 # recipe_pcrglobwb.yml
 ---
 documentation:
-  description: Pre-processes climate data for use in the PCR-GLOBWB hydrological model.
+  description: |
+    Pre-processes climate data for use in the PCR-GLOBWB hydrological model.
+
   authors:
     - aerts_jerom
     - andela_bouwe
     - drost_niels
     - camphuijsen_jaro
     - alidoost_sarah
     - kalverla_peter
-  projects: ['ewatercycle']
-  references: ['acknow_project', 'sutanudjaja2018gmd']
+
+  projects:
+    - ewatercycle
+
+  references:
+    - sutanudjaja2018gmd
```

Comment:
You lost acknow_project here

Reply:
Though that is actually not a reference, but a request to acknowledge, so maybe we need to think a bit about how to improve this in the future

Reply:
`acknow_project` was introduced for cases where no other references were available, which is not case here.

Reply:
That's probably never the case, because all our recipes are based on scientific work, right? Only the example recipes wouldn't have references.

Reply:
Actually there are 5 recipes without an actual reference: `recipe_consecdrydays.yml`, `recipe_ocean_Landschuetzer2016.yml`, `recipe_ocean_multimap.yml`, `recipe_shapeselect.yml` and `recipe_spei.yml.

Reply:
Maybe references just need to be added?

Reply:
Maybe yes, but not all our diagnostics are based on published work, or the paper is not yet ready.

Reply:
I think it would be fine to have no reference if no reference is available.